collector/lib/CollectorConfig.h

```diff
@@ -75,7 +78,19 @@ class CollectorConfig {
   bool IsProcessesListeningOnPortsEnabled() const { return enable_processes_listening_on_ports_; }
   bool ImportUsers() const { return import_users_; }
   bool CollectConnectionStatus() const { return collect_connection_status_; }
-  bool EnableExternalIPs() const { return enable_external_ips_; }
+
+  // EnableExternalIPs will check for the existence
+  // of a runtime configuration, and defer to that value
+  // otherwise, we rely on the feature flag (env var)
+  bool EnableExternalIPs() const {
```

Comment:
This function could be called `IsExternalIPsEnabled` or `GetExternalIPsEnabled` to make it clear that this checks the if external IPs should be enabled and it doesn't actually set any values. However, your naming is consistent with the rest of the code, so you can keep it.

Reply:
Resolving as the naming here is consistent with the rest of the code.